.github/workflows/latest_release.yaml

```diff
@@ -0,0 +1,58 @@
+name: Latest Release
+
+on:
+  push:
+    branches:
+    - "main"
+    paths-ignore:
+    - "**/*.png"
+
+jobs:
+  latest-release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Build i2gw latest multiarch binaries
+      run: | 
+        make build-multiarch
+        tar -zcvf i2gw_latest_linux_amd64.tar.gz _output/linux/amd64/
+        tar -zcvf i2gw_latest_linux_arm64.tar.gz _output/linux/arm64/
+        tar -zcvf i2gw_latest_darwin_amd64.tar.gz _output/darwin/amd64/
+        tar -zcvf i2gw_latest_darwin_arm64.tar.gz _output/darwin/arm64/
+
+    # Ignore the error when we delete the latest release, it might not exist.
+    - name: Delete the Latest Release
```

Comment:
This seems very dangerous, can we leave this out?

Reply:
IMHO, I suggest a latest release for users to use latest binary. This workflow will auto release every time we merged a new commit. Delete / Create Latest Tag and Release. This is how we do in EG.

Reply:
`latest` == `main` is okay as long as we make sure that people understand that `latest` != "latest _stable_".

Reply:
I have removed it, I think it could be discussed/added later and if anyone else requests for it too. Do not want it to be a blocker for landing.